otx/recipes/stages/classification/incremental.yaml

```diff
@@ -22,3 +22,5 @@ evaluation:
 task_adapt:
   type: "mpa"
   op: "REPLACE"
+
+channel_last: True
```

Comment:
Why don't we add this more general recipe so that selfsl and semisl also can be faster

Reply:
Good suggestion, I'll apply

Reply:
7b1445821eaf7bbd0c2ea71bb8803665f9e7e24c